packages/xo-web/src/common/xo/index.js

```diff
@@ -390,13 +390,19 @@ export const subscribeNotifications = createSubscription(async () => {
   if (
     process.env.XOA_PLAN === 5 ||
     xoaUpdaterState === 'disconnected' ||
-    xoaUpdaterState === 'error' ||
-    xoaUpdaterState === 'registerNeeded'
+    xoaUpdaterState === 'error'
   ) {
     return []
   }
 
-  const notifications = await updater._call('getMessages')
+  let notifications
+  try {
+    notifications = await updater._call('getMessages')
+  } catch (err) {
+    if (err.message !== 'Not registered') {
```

Comment:
I think we should always log the error and return `[]`.

Reply:
That would generate a lot of logs, though.

Reply:
Then maybe not logging the error and always succeed?

Reply:
Even when the error is not `Not registered`?